clusterloader2/state/namespaces.go

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
I'd suggest moving this package into `pkg/` (like we were discussing earlier) - as this is internal to the implementation.

Reply:
ok